logistics/scripts/Invoke-SdkGen.ps1

```diff
@@ -44,13 +45,13 @@ function Invoke-SdkGen {
     $apiMetadataUrl = ($apiUrl + "/metadata?sdk=true")
     $teamCityParameters = Get-TeamCityParameters
     $buildConfiguration = Get-ValueOrDefault $teamCityParameters['msbuild.buildConfiguration'] 'Debug'
```

Comment:
it's passing debug. i think we need to pass release for this buildconfiguration parameter also from github actions